janusgraph-examples/example-cassandra/README.md

```diff
@@ -0,0 +1,86 @@
+# Cassandra Thrift Storage, Elasticsearch Index
+
+## About Cassandra and Elasticsearch
+
+[Apache Cassandra](http://cassandra.apache.org/) is a distributed database
+designed for scalability and high availability. Cassandra supports two
+protocols for communications, Thrift (legacy RPC protocol) and CQL (native
+protocol). Depending on the Cassandra version, Thrift may not be started by
+default. Make sure that [Thrift is started](http://docs.datastax.com/en/cassandra/2.1/cassandra/tools/toolsStatusThrift.html)
+when using this example.
+
+[Elasticsearch](https://www.elastic.co/products/elasticsearch) is a scalable,
+distributed search engine.
+
+> Check the JanusGraph [version compatibility](http://docs.janusgraph.org/latest/version-compat.html)
+to ensure you select versions of Cassandra and Elasticsearch compatible with
+this JanusGraph release.
+
+## JanusGraph Configuration
+
+* [`jgex-cassandra.properties`](conf/jgex-cassandra.properties) contains the
+Cassandra and Elasticsearch server locations. By providing different values
+for `storage.cassandra.keyspace` and `index.jgex.index-name`, you can store
+multiple graphs on the same Cassandra and Elasticsearch servers. Refer to
+the JanusGraph [configuration reference](http://docs.janusgraph.org/latest/config-ref.html)
+for additional properties.
+
+* [`logback.xml`](conf/logback.xml) configures logging with [Logback](https://logback.qos.ch/),
+which is the logger used by Cassandra. The example configuration logs to the
+console and adjusts the logging level for some noisier packages. Refer to
+the Logback [manual](https://logback.qos.ch/manual/index.html) for additional
+details.
+
+## Run the Example
+
+Use [Apache Maven](http://maven.apache.org/) and the [exec-maven-plugin](http://www.mojohaus.org/exec-maven-plugin/java-mojo.html)
+to pull in the required jar files onto the runtime classpath.
+
+```
+mvn exec:java -Dexec.mainClass="org.janusgraph.example.JanusGraphApp" -Dexec.args="conf/jgex-cassandra.properties" -Dlogback.configurationFile="conf/logback.xml"
+```
+
+## Drop the Graph
+
+Make sure to stop the application before dropping the graph.
+
+### Drop the cassandra keyspace
+
+The keyspace name matches the value of `storage.cassandra.keyspace`
+
+```
+$ cd $CASSANDRA_HOME
+$ bin/cqlsh -e 'describe keyspaces'
+system_traces  system  jgex
+$ bin/cqlsh -e 'drop keyspace jgex'
+$ bin/cqlsh -e 'describe keyspaces'
+system_traces  system
+```
```

Comment:
You could also use the core factory API (`JanusGraphFactory.drop(Graph)`) to drop the keyspace. The drop method is fully supported on all storage backends (Cassandra/CQL, HBase, BerkeleJE) and indexing backends with the exception of Solr (HTTP client).

Reply:
Good call, I don't think that was implemented yet when I first started working on these examples.

Reply:
Note it currently throws an exception when using Solr in HTTP mode but there's a proposal to lower that to an error message in #534. Either way the storage backend is still dropped.